ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/StaticQueryExecutor.java

```diff
@@ -117,21 +117,7 @@ public static void validate(
       final KsqlExecutionContext executionContext,
       final ServiceContext serviceContext
   ) {
-    final Query queryStmt = statement.getStatement();
-
-    if (!statement.getConfig().getBoolean(KsqlConfig.KSQL_PULL_QUERIES_ENABLE_CONFIG)) {
-      throw new KsqlRestException(
-          Errors.badStatement(
-              "Pull queries are disabled on this KSQL server - please set "
-                  + KsqlConfig.KSQL_PULL_QUERIES_ENABLE_CONFIG + "=true to enable this feature. "
-                  + "If you intended to issue a push query, resubmit the query with the "
-                  + "EMIT CHANGES clause.",
-              statement.getStatementText()));
-    }
-
-    if (!queryStmt.isStatic()) {
```

Comment:
should we also move this check into the executor?

Reply:
Yep. can do.